recipes/sdl/all/conanfile.py

```diff
@@ -170,6 +173,13 @@ def package_id(self):
             del self.info.options.sdl2main
 
     def build_requirements(self):
+        if self.settings.os == "Macos" and tools.is_cross_building(self):
```

Comment:
My conan is getting rusty :wink: 
https://docs.conan.io/en/latest/reference/tools.html#tools-cross-building
```suggestion
        if self.settings.os == "Macos" and tools.cross_building(self):
```